File: bezbednost/src/main/java/bezbednost/dto/OtherCertificate.java
package bezbednost.dto;


public class OtherCertificate extends Certificate {
	
	private String issuerAlias;
	private boolean isCA;
	
	
	public OtherCertificate() {
		
	}

	public OtherCertificate(String ksName, String privateKeyPassword, String issuerAlias, String commonName,
			String alias, String orgName, String orgUnit, String country, String email, String serialNum,
			String validity, String purpose, boolean isCA) {
		super(commonName, alias, orgName, orgUnit, country, email,
			    serialNum, ksName, privateKeyPassword, validity, purpose);
		this.issuerAlias = issuerAlias;
		this.isCA = isCA;
	}

	public String getIssuerAlias() {
		return issuerAlias;
	}

	public boolean isCA() {
		return isCA;
	}

	public void setIssuerAlias(String issuerAlias) {
		this.issuerAlias = issuerAlias;
	}

	public void setIsCA(boolean isCA) {
		this.isCA = isCA;
	}	
	
}

File: nistagram-front/src/app/model/other-certificate.ts
export class OtherCertificate {
    public issuerAlias: string;
    public commonName : string;
    public alias : string;
    public orgName: string;
    public orgUnit: string;
    public country: string;
    public email: string;
    public serialNum: string;
    public validity: number;
    public purpose: string;
    public isCA: boolean;

    constructor(issuerAlias: string, commonName: string, alias: string, orgName: string, orgUnit: string,
        country: string, email: string, serialNum: string, validity: number, purpose: string, isCA: boolean) {
            this.issuerAlias = issuerAlias;
            this.commonName = commonName;
            this.alias = alias;
            this.orgName = orgName;
            this.orgUnit = orgUnit;
            this.country = country;
            this.email = email;
            this.serialNum = serialNum;
            this.validity = validity;
            this.purpose = purpose;
            this.isCA = isCA;
    }
}